src/core/autofill.cc

```diff
@@ -124,6 +124,9 @@ AutoFill::Create(
   const Platform platform = GetPlatform(config.platform());
 #endif
 
+  // If model operations is specified, use it to set the session options for ONNX.
+  auto model_ops = config.model_operations();
```

Comment:
This is only needed in the ONNX case so define it only where needed below.

Reply:
Will do